File: java/warmup-1/monkeyTrouble.java
/* We have two monkeys, a and b, and the parameters aSmile and bSmile indicate 
 * if each is smiling. We are in trouble if they are both smiling or if 
 * neither of them is smiling. Return true if we are in trouble.
 */
public boolean monkeyTrouble(boolean aSmile, boolean bSmile) {
    return aSmile == bSmile;
}

File: python/warmup-1/monkey_trouble.py
# We have two monkeys, a and b, and the parameters a_smile and b_smile 
# indicate if each is smiling. We are in trouble if they are both smiling or 
# if neither of them is smiling. Return True if we are in trouble.
def monkey_trouble(a_smile, b_smile):
    return a_smile == b_smile
